Algorithm.CSharp/BasicTemplateAlgorithm.cs

```diff
@@ -119,4 +119,4 @@ public override void OnData(Slice data)
             {"OrderListHash", "491919591"}
         };
     }
-}
+}
```

Comment:
nit - might be a nice git exercise to remove this file 😃

Reply:
Necessary change, removes /r/n from end of file. Important for test that uploads code and then reads the code in the cloud to compare.